tests/styles/test_oxford_comma.py

```
@@ -117,3 +117,49 @@ def test_oxford_comma_allows_parenthetical_with_serial_comma(
     diags = concordat_vale.lint(text)
 
     assert diags == [], "expected no diagnostics when comma precedes the conjunction"
+
+
+def test_oxford_comma_ignores_with_or_without_clause(
+    concordat_vale: Valedate,
+) -> None:
+    """Phrases like 'with or without fee' are not serial lists."""
+    text = textwrap.dedent(
+        """
+        ISC Licence — because that’s how we roll. You’re free to use, copy, modify, and
```

Ruff (0.14.7)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)